HuduAPI/Private/Invoke-HuduRequest.ps1

```diff
@@ -61,6 +61,13 @@ function Invoke-HuduRequest {
         'x-api-key' = (New-Object PSCredential 'user', $HuduAPIKey).GetNetworkCredential().Password;
     }
 
+    if (($null -ne $Int_HuduCustomHeaders) -and ($Int_HuduCustomHeaders.Count -gt 0)) {
```

Comment:
```suggestion
    if (($Script:Int_HuduCustomHeaders | Measure-Object).count -gt 0){
```
This way it makes sure it reads it from the script scope and will avoid problems if there is only 1 item in Int_HuduCustomHeaders